frontend/openchat-agent/src/utils/caching.ts

```diff
@@ -48,7 +48,7 @@ import type { CryptocurrencyContent } from "openchat-shared";
 import type { PrizeContent } from "openchat-shared";
 import type { P2PSwapContent } from "openchat-shared";
 
-const CACHE_VERSION = 103;
+const CACHE_VERSION = 104;
```

Comment:
So there was no way to fix this more surgically?

Reply:
I think I'd rather tackle that when there is no immediate urgency because it's something that we _really_ don't want to fuck up.